src/pke/lib/scheme/bgvrns/bgvrns-parametergeneration.cpp

```diff
@@ -111,14 +111,14 @@ BGVNoiseEstimates ParameterGenerationBGVRNS::computeNoiseEstimates(
         }
         int relinBase       = pow(2.0, digitSize);
         int modSizeEstimate = DCRT_MODULUS::MAX_SIZE;
-        int numWindows      = floor(modSizeEstimate / log(relinBase)) + 1;
+        int numWindows      = floor(modSizeEstimate / digitSize) + 1;
```

Comment:
std::floor doesn't make any sense here as this is a division of 2 unsigned integers (even one of them declared as "int").

Reply:
Agreed